docs/source/commands.rst

```diff
@@ -1,3 +1,5 @@
+.. _management-commands
```

Comment:
Does this work? IMO, we need to add ":":

```rst
.. _management-commands:
```

Reply:
You're right. I missed that.